etna/core/models/mixins.py

```diff
@@ -25,3 +29,47 @@ class ContentWarningMixin(models.Model):
 
     class Meta:
         abstract = True
+
+
+class NewLabelMixin(models.Model):
+    """Mixin to allow editors to toggle 'new' label to be applied on-publish"""
+
+    mark_new_on_next_publish = models.BooleanField(
+        verbose_name="mark this page as 'new' when published",
+        default=True,
+        help_text="Mark this as true before publishing, to display the 'new label'",
```

Comment:
This help text feels a bit redundant now, and we're using "Mark" to mean 'check this box' and 'mark the page', which could be confusing. Maybe just ditch the help text and add something later if editors don't think it's clear enough?